common/unsafe/src/main/java/org/apache/spark/unsafe/types/CalendarInterval.java

```diff
@@ -41,7 +41,7 @@
  * they are two separated fields from microseconds. One month may be equal to 28, 29, 30 or 31 days
  * and one day may be equal to 23, 24 or 25 hours (daylight saving).
  *
- * @since 1.5.0
+ * @since 3.0.0
  */
 @Unstable
```

Comment:
BTW, are we going to expose this as an API? Technically, this is under internal package which isn't documented:
https://github.com/apache/spark/blob/master/project/SparkBuild.scala#L828

Reply:
Let me open a PR after a quick test.

Reply:
+1